### DISCUSSION TITLE
Workspaces color

### DISCUSSION THREAD

**Discussion by UltraBlackLinux** [Alexays/Waybar#1949] [Q&A] [answered]
Hey there,
I'm having trouble changing the color of the workspace widget's text. It always stays white: ![screenshot-20230115-143524](https://user-images.githubusercontent.com/62404294/212543876-a486c448-9bbc-4dc5-be70-9e3ad93adc36.png)

Changing it's `color` does not work for me, what is supposed to be used?

Thanks!

**Comment by miranyx** [comment, marked as answer]
Try this in your style.css, this should make the text color yellow.

```css
#workspaces button {
  color: #e1e802;
}
```

**Reply by UltraBlackLinux** [reply]
oh thank you so much! That works perfectly!
